chat-widget/src/components/livechatwidget/common/startChat.ts

```diff
@@ -78,11 +78,11 @@ const setPreChatAndInitiateChat = async (chatSDK: any, dispatch: Dispatch<ILiveC
     //Initiate start chat
     dispatch({ type: LiveChatWidgetActionType.SET_CONVERSATION_STATE, payload: ConversationState.Loading });
     const optionalParams: StartChatOptionalParams = { isProactiveChat };
-    await initStartChat(chatSDK, props?.chatConfig, props?.getAuthToken, dispatch, setAdapter, optionalParams);
+    await initStartChat(chatSDK, props?.chatConfig, props?.getAuthToken, dispatch, setAdapter, props?.controlProps?.hideErrorUIPane, optionalParams);
 };
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-const initStartChat = async (chatSDK: any, chatConfig: ChatConfig | undefined, getAuthToken: ((authClientFunction?: string) => Promise<string | null>) | undefined, dispatch: Dispatch<ILiveChatWidgetAction>, setAdapter: any, params?: StartChatOptionalParams, persistedState?: any) => {
+const initStartChat = async (chatSDK: any, chatConfig: ChatConfig | undefined, getAuthToken: ((authClientFunction?: string) => Promise<string | null>) | undefined, dispatch: Dispatch<ILiveChatWidgetAction>, setAdapter: any, hideErrorUIPane?: boolean, params?: StartChatOptionalParams, persistedState?: any) => {
```

Comment:
This method's argument is getting out of control. Could you please group all the argument under one interface and change all the occurrences? Please help validate the occurrences with `initStartChat`

Reply:
Most of the arguments here can be extracted from the LiveChatWidget props
Should I update the arguments to pass the whole props it can be reduced that way